### ISSUE TITLE
triage failing examples

### ISSUE THREAD

**Issue by ryanleecode** [paritytech/capi#761]
[first_ten_keys.ts]
```
vscode ➜ /workspaces/capi (example-tool-testing-2) $ deno task run examples/first_ten_keys.ts 
Task run deno task capi -- deno run -A -r=http://localhost:4646/ "examples/first_ten_keys.ts"
Task capi deno run -A main.ts --dbg "--" "deno" "run" "-A" "-r=http://localhost:4646/" "examples/first_ten_keys.ts"
Reusing existing Capi server at "http://localhost:4646/"
error: Uncaught (in promise) Unhandled {
  value: ScaleAssertError: !(value instanceof Uint8Array)
    at AssertState.instanceof (https://deno.land/x/scale@v0.11.0/common/assert.ts:37:13)
    at Codec._assert (https://deno.land/x/scale@v0.11.0/codecs/array.ts:104:24)
    at Codec._assert (file:///workspaces/capi/crypto/hashers.ts:52:14)
    at Codec._assert (file:///workspaces/capi/frame_metadata/key_codecs.ts:25:12)
    at Module.assert (https://deno.land/x/scale@v0.11.0/common/codec.ts:140:9)
    at RunMap.fn (file:///workspaces/capi/fluent/CodecRune.ts:8:9)
    at RunMap._evaluate (file:///workspaces/capi/rune/ValueRune.ts:153:40)
    at async RunMap.evaluate (file:///workspaces/capi/rune/Rune.ts:226:14)
    at async RunThrows._evaluate (file:///workspaces/capi/rune/ValueRune.ts:184:14)
    at async RunThrows.evaluate (file:///workspaces/capi/rune/Rune.ts:226:14)
}
```

[size_tree.ts]
```
vscode ➜ /workspaces/capi (example-tool-testing-2) $ deno task run examples/size_tree.ts
Task run deno task capi -- deno run -A -r=http://localhost:4646/ "examples/size_tree.ts"
Task capi deno run -A main.ts --dbg "--" "deno" "run" "-A" "-r=http://localhost:4646/" "examples/size_tree.ts"
Capi server listening at "http://localhost:4646/"
error: Uncaught (in promise) Unhandled {
  value: ScaleAssertError: !(value instanceof Uint8Array)
    at AssertState.instanceof (https://deno.land/x/scale@v0.11.0/common/assert.ts:37:13)
    at Codec._assert (https://deno.land/x/scale@v0.11.0/codecs/array.ts:104:24)
    at Codec._assert (file:///workspaces/capi/crypto/hashers.ts:52:14)
    at Codec._assert (file:///workspaces/capi/frame_metadata/key_codecs.ts:58:14)
    at Codec._assert (file:///workspaces/capi/frame_metadata/key_codecs.ts:25:12)
    at Module.assert (https://deno.land/x/scale@v0.11.0/common/codec.ts:140:9)
    at RunMap.fn (file:///workspaces/capi/fluent/CodecRune.ts:8:9)
    at RunMap._evaluate (file:///workspaces/capi/rune/ValueRune.ts:153:40)
    at async RunMap.evaluate (file:///workspaces/capi/rune/Rune.ts:226:14)
    at async RunThrows._evaluate (file:///workspaces/capi/rune/ValueRune.ts:184:14)
}
```

[virtual_multisig.ts]
```
vscode ➜ /workspaces/capi (example-tool-testing-2) $ deno task run examples/virtual_multisig.ts
Task run deno task capi -- deno run -A -r=http://localhost:4646/ "examples/virtual_multisig.ts"
Task capi deno run -A main.ts --dbg "--" "deno" "run" "-A" "-r=http://localhost:4646/" "examples/virtual_multisig.ts"
Capi server listening at "http://localhost:4646/"
Proxy creation: ready
Proxy creation: { inBlock: "0x026258f1bc883b0dce7f2c8d87dc70b4fd4c8ce0700eb08306226caa1c09ca6f" }
Proxy creation: { finalized: "0x026258f1bc883b0dce7f2c8d87dc70b4fd4c8ce0700eb08306226caa1c09ca6f" }
error: Uncaught (in promise) Unhandled {
  value: ScaleAssertError: !("type" in value.call.value.calls[0].value.dest)
    at AssertState.key (https://deno.land/x/scale@v0.11.0/common/assert.ts:45:13)
    at Codec._assert (https://deno.land/x/scale@v0.11.0/codecs/union.ts:57:32)
    at Codec._assert (https://deno.land/x/scale@v0.11.0/codecs/object.ts:16:14)
    at Codec._assert (https://deno.land/x/scale@v0.11.0/codecs/object.ts:79:16)
    at Codec._assert (https://deno.land/x/scale@v0.11.0/codecs/object.ts:79:16)
    at Codec._assert (https://deno.land/x/scale@v0.11.0/codecs/union.ts:62:76)
    at Codec._assert (https://deno.land/x/scale@v0.11.0/codecs/object.ts:16:14)
    at Codec._assert (https://deno.land/x/scale@v0.11.0/codecs/object.ts:79:16)
    at Codec._assert (https://deno.land/x/scale@v0.11.0/codecs/object.ts:79:16)
    at Codec._assert (https://deno.land/x/scale@v0.11.0/codecs/union.ts:62:76)
}
```

[polkadot_js_signer.ts]
```
vscode ➜ /workspaces/capi (example-tool-testing-2) $ deno task run examples/polkadot_js_signer.ts
Task run deno task capi -- deno run -A -r=http://localhost:4646/ "examples/polkadot_js_signer.ts"
Task capi deno run -A main.ts --dbg "--" "deno" "run" "-A" "-r=http://localhost:4646/" "examples/polkadot_js_signer.ts"
Capi server listening at "http://localhost:4646/"
error: Uncaught (in promise) Unhandled {
  value: ServerError: Invalid Transaction
    at RunMap.fn (file:///workspaces/capi/fluent/ConnectionRune.ts:53:32)
    at RunMap._evaluate (file:///workspaces/capi/rune/ValueRune.ts:153:40)
    at async RunMap.evaluate (file:///workspaces/capi/rune/Rune.ts:226:14)
    at async RunThrows._evaluate (file:///workspaces/capi/rune/ValueRune.ts:184:14)
    at async RunThrows.evaluate (file:///workspaces/capi/rune/Rune.ts:226:14)
    at async Promise.all (index 0)
    at async RunLs.evaluate (file:///workspaces/capi/rune/Rune.ts:226:14)
    at async RunMap._evaluate (file:///workspaces/capi/rune/ValueRune.ts:151:20)
    at async RunMap.evaluate (file:///workspaces/capi/rune/Rune.ts:226:14)
    at async RunFilter._evaluate (file:///workspaces/capi/rune/ValueRune.ts:264:21)
Caused by Invalid Transaction
}
```

[multisig_pure_proxy_stash.ts]
```
vscode ➜ /workspaces/capi (example-tool-testing-2) $ deno task run examples/multisig_pure_proxy_stash.ts
Task run deno task capi -- deno run -A -r=http://localhost:4646/ "examples/multisig_pure_proxy_stash.ts"
Task capi deno run -A main.ts --dbg "--" "deno" "run" "-A" "-r=http://localhost:4646/" "examples/multisig_pure_proxy_stash.ts"
Capi server listening at "http://localhost:4646/"
Fund Multisig: ready
Fund Multisig: { inBlock: "0xec120d19da761bcf121ac73ff998732488e3f8805fd24811ac87b8bbc529a870" }
Fund Multisig: { finalized: "0xec120d19da761bcf121ac73ff998732488e3f8805fd24811ac87b8bbc529a870" }
error: Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'when')
        value = value[key]
                     ^
    at RunMap.fn (file:///workspaces/capi/rune/ValueRune.ts:53:22)
    at RunMap._evaluate (file:///workspaces/capi/rune/ValueRune.ts:153:40)
    at async RunMap.evaluate (file:///workspaces/capi/rune/Rune.ts:226:14)
    at async RunCaptureUnhandled._evaluate (file:///workspaces/capi/rune/Rune.ts:370:14)
    at async RunCaptureUnhandled.evaluate (file:///workspaces/capi/rune/Rune.ts:226:14)
    at async Promise.all (index 6)
    at async RunLs.evaluate (file:///workspaces/capi/rune/Rune.ts:226:14)
    at async RunMap._evaluate (file:///workspaces/capi/rune/ValueRune.ts:151:20)
    at async RunMap.evaluate (file:///workspaces/capi/rune/Rune.ts:226:14)
    at async Promise.all (index 1)
```

[entry_page.ts]
```
vscode ➜ /workspaces/capi (example-tool-testing-2) $ deno task run examples/entry_page.ts               
Task run deno task capi -- deno run -A -r=http://localhost:4646/ "examples/entry_page.ts"
Task capi deno run -A main.ts --dbg "--" "deno" "run" "-A" "-r=http://localhost:4646/" "examples/entry_page.ts"
Capi server listening at "http://localhost:4646/"
error: Uncaught Unhandled {
  value: ScaleAssertError: typeof value !== "number"
    at AssertState.typeof (https://deno.land/x/scale@v0.11.0/common/assert.ts:25:13)
    at Codec._assert (https://deno.land/x/scale@v0.11.0/codecs/int.ts:37:20)
    at Codec._assert (file:///workspaces/capi/crypto/hashers.ts:52:14)
    at Codec._assert (file:///workspaces/capi/frame_metadata/key_codecs.ts:25:12)
    at Module.assert (https://deno.land/x/scale@v0.11.0/common/codec.ts:140:9)
    at RunMap.fn (file:///workspaces/capi/fluent/CodecRune.ts:8:9)
    at RunMap._evaluate (file:///workspaces/capi/rune/ValueRune.ts:153:40)
    at async RunMap.evaluate (file:///workspaces/capi/rune/Rune.ts:226:14)
    at async RunThrows._evaluate (file:///workspaces/capi/rune/ValueRune.ts:184:14)
    at async RunThrows.evaluate (file:///workspaces/capi/rune/Rune.ts:226:14)
}
```

**Comment by ryanleecode** [comment]
a lot of these seem to be codec errors. @tjjfvi any quick guesses as to what the root cause could be?

**Comment by tjjfvi** [comment]
Some of these should be fixed as of #768. @ryanleecode could you rerun and post which ones are still failing?

**Comment by ryanleecode** [comment]
I have these failing:

`multisig_pure_proxy_stash.ts`
`polkadot_js_signer.ts`
`metadata.ts`
`virtual_multisig.ts`
`size_tree.ts`
